pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -829,6 +829,15 @@ in
     nvimRequireCheck = "crates";
   };
 
+  cpsell-nvim = super.cspell-nvim.overrideAttrs {
```

Comment:
```suggestion
  cspell-nvim = super.cspell-nvim.overrideAttrs {
```

Reply:
This is why, the overrides aren't being passed to cspell, they are building a new plugin cpsell